### DISCUSSION TITLE
coc.nvim clangd不会跳转了？

### DISCUSSION THREAD

**Discussion by ChenDaxia-Sudo** [neoclide/coc.nvim#4563] [Q&A]
当前打开vim不会出现clangd在解析中的提示，以及使用跳转快捷键完全没有任何的反应

![image](https://user-images.githubusercontent.com/52773833/224474755-59662d8f-80ee-4c70-9fbd-4aaf13e41d52.png)

**Comment by fannheyward** [comment]
应该是你的配置有问题，看一下 trace log。

另外不建议同时使用 clangd 和 ccls。

**Reply by ChenDaxia-Sudo** [reply]
非常感谢回答~
> 应该是你的配置有问题，看一下 trace log。


请教下，trace log是指CocOpenLog的结果吗

![image](https://user-images.githubusercontent.com/52773833/224476643-35230be6-6b74-4d29-a647-26ec2fc16a49.png)

> 
> 另外不建议同时使用 clangd 和 ccls。

没有配置ccls来着

![image](https://user-images.githubusercontent.com/52773833/224476668-b73dd253-5eaf-4e0e-ae22-6308c70da035.png)

**Reply by fannheyward** [reply]
https://github.com/neoclide/coc.nvim/wiki/Debug-language-server#using-output-channel
